### PR TITLE
Update markets for Berlin (published 10.06.2016).

### PR DESCRIPTION
**Pull request by johnjohndoe**

### CODE REVIEW

File: preprocessing/berlin/raw/markets-berlin.json [hunk 1]
```diff
@@ -718,10 +718,10 @@
...
+                "description": "<strong>Wochenmarkt Am Kaufhof Ostbahnhof\n(Erich-Steinfurth-Straße)</strong><br />Mo - Fr\nSa<br />10:00 - 19:00\n09:00 - 16:00<br /> <a href=\"http://www.berlin.de/sen/wirtschaft/service/maerkte-feste/wochen-troedelmaerkte/index.php/detail/45\">Mehr...</a><br /><br /><strong>design.Börse_Berlin in Galeria Kaufhof am Ostbahnhof, Erich-Steinfurth-Straße</strong><br />22. - 24.01.2016<br />Fr 16:00-20:00\nSa 12:00-20:00\nSo 13:00-18:00<br /> <a href=\"http://www.berlin.de/sen/wirtschaft/service/maerkte-feste/wochen-troedelmaerkte/index.php/detail/61\">Mehr...</a>",
                 "href": "http://www.berlin.de/sen/wirtschaft/service/maerkte-feste/wochen-troedelmaerkte/index.php/detail/45",
                 "id": "http://www.berlin.de/sen/wirtschaft/service/maerkte-feste/wochen-troedelmaerkte/index.php/detail/45",
-                "title": "Wochenmarkt Am Kaufhof Ostbahnhof\n(Erich-Steinfurth-Straße)"
+                "title": "Mehrerer Einträge..."
```
Comment (k-nut) [inline preprocessing/berlin/raw/markets-berlin.json]: @johnjohndoe didn't you make sure in the last commit that we do not have `Mehrere Einträge` titles anymore?

Comment (johnjohndoe) [inline preprocessing/berlin/raw/markets-berlin.json]: This is the *raw* file I download from the website.

Comment (k-nut) [inline preprocessing/berlin/raw/markets-berlin.json]: Ah stupid me. So forget everything I said ;)

Comment (johnjohndoe) [inline preprocessing/berlin/raw/markets-berlin.json]: So ... how about a review then?

**Comment by k-nut** [comment]
Looks good to me! 👍 want me to merge it?

**Comment by johnjohndoe** [comment]
Yes, please go ahead (please rebase if master has grown meanwhile).

**Comment by johnjohndoe** [comment]
Yes, please.

**Pull request merged**